packages/@ourworldindata/grapher/src/entitySelector/EntitySelector.tsx

```diff
@@ -720,6 +777,7 @@ export class EntitySelector extends React.Component<{
                 name: entityName,
                 sortColumnValues: {},
                 alternativeNames: getRegionAlternativeNames(entityName, langs),
+                regionInfo: getRegionByName(entityName),
```

Comment:
You can use this property now also in `entitiesByRegionType`, getting rid of the `getRegionByName` call in there.